conceptual.md

```diff
@@ -3,22 +3,35 @@
 Answer the following questions below:
 
 - What is PostgreSQL?
+  * An open source object relational database management system
 
 - What is the difference between SQL and PostgreSQL?
+  * PostgreSQL is an advanced version of SQL which provides support to different functions of SQL like foreign keys, subqueries, triggers, and different user-defined types and functions.
 
 - In `psql`, how do you connect to a database?
+```
+$ psql db_name
+```
 
 - What is the difference between `HAVING` and `WHERE`?
+  * WHERE filters row given a condition. HAVING filters a group (GROUP BY) give a condition.
 
 - What is the difference between an `INNER` and `OUTER` join?
+  * INNER join returns only the rows that match a given condition in both tables whereas OUTER joins can return non-matching rows as well.
 
 - What is the difference between a `LEFT OUTER` and `RIGHT OUTER` join?
+  * LEFT OUTER returns all rows from the first table as well as rows from the second table that match the given condition. RIGHT OUTER returns all rows from the second table as well as rows from the first table that match the given condition.
 
 - What is an ORM? What do they do?
+  * An ORM is a library for object relational mapping that allows you to interact with a database like PostgreSQL in your language of choice such as Python.
 
 - What are some differences between making HTTP requests using AJAX 
   and from the server side using a library like `requests`?
+    * Server side works with APIs that have a same-origin policy.
```

Comment:
Can you expand on this? Service side _exposes_ APIs that have a same-origin policy, typically used by a frontend requester.